### PR TITLE
Added entries to travis and updated deps

### PR DESCRIPTION
**Pull request by hugofqueiros**
Build on travis and tests are passing

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules
 /tmp
+npm-debug.log*
```
Comment (SamVerschueren) [inline .gitignore]: remove

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+.idea
```
Comment (SamVerschueren) [inline .gitignore]: remove

File: .travis.yml [hunk 3]
```diff
@@ -4,3 +4,7 @@ node_js:
   - 'iojs'
   - '0.12'
   - '0.10'
+  - "4.4"
```
Comment (SamVerschueren) [inline .travis.yml]: Use this list instead

```
  - '5'
  - '4'
  - '0.12'
  - '0.10'
```

**Comment by hugofqueiros** [comment]
done thanks for the comments

File: package.json [hunk 4]
```diff
@@ -34,11 +34,11 @@
...
     "gulp-util": "^3.0.0",
     "through2": "^2.0.0",
-    "vulcanize": "^1.9.1"
+    "vulcanize": "^1.14.7"
   },
   "devDependencies": {
     "mkdirp": "^0.5.0",
     "mocha": "*",
-    "rimraf": "^2.2.8"
+    "rimraf": "^2.5.2"
```
Comment (sindresorhus) [inline package.json]: Not needed: https://nodesource.com/blog/semver-tilde-and-caret

File: .travis.yml [hunk 5]
```diff
@@ -4,3 +4,7 @@ node_js:
   - 'iojs'
   - '0.12'
   - '0.10'
+  - "4"
+  - "5"
```
Comment (sindresorhus) [inline .travis.yml]: Please follow the existing style when doing a PR>

Comment (SamVerschueren) [inline .travis.yml]: Odd, I already commented on that...

**Pull request closed without merging**